api/test/gemComparison.test.js

```diff
@@ -0,0 +1,28 @@
+import fetch from "node-fetch";
+
+describe("GEM comparison", () => {
+  test("the comparison should have reactions data", async () => {
+    const res = await fetch(
+      `${API_BASE}/compare?models=[%7B"model":"FruitflyGem","version":"1_1_0"%7D,%7B"model":"HumanGem","version":"1_10_0"%7D]`
+    );
+
+    const { Reaction } = await res.json();
+    expect(Reaction[0].FruitflyGem).toBeGreaterThanOrEqual(
+      Reaction[2].FruitflyGem
+    );
+    expect(Reaction[1].HumanGem).toBeGreaterThanOrEqual(Reaction[2].HumanGem);
+  });
+
+  test("the comparison details should have lists of unique reactions and metabolites", async () => {
+    const res = await fetch(
+      `${API_BASE}/comparison-details?model=%7B"model":"FruitflyGem","version":"1_1_0"%7D&models=[%7B"model":"HumanGem","version":"1_10_0"%7D]`
+    );
+
+    const {
+      details: { CompartmentalizedMetabolite, Reaction },
+    } = await res.json();
+
+    expect(CompartmentalizedMetabolite.unique.length).toBeGreaterThan(0);
+    expect(Reaction.unique.length).toBeGreaterThan(0);
```

Comment:
maybe I'm missing something here, but I assume `Reaction.unique.length` won't actually check if all elements are unique? the description of the test ("...should have lists of unique reactions and metabolites") seems to indicate that there should be no duplicated elements.

Reply:
Ah, I could have probably worded the description better. I was looking at this part of the website as I was writing it. Do you think it makes sense in this context?

<img width="504" alt="image" src="https://user-images.githubusercontent.com/423498/148977608-a351b1f7-babe-412d-9f23-85c0646c417f.png">

This is the response data:

<img width="453" alt="image" src="https://user-images.githubusercontent.com/423498/148977866-524744a0-79b0-4a19-93ba-34bae33b3c57.png">

Reply:
ah, yes, now I get it :D 
Maybe a rephrase would prevent future confusion?
`"the comparison details should have lists of reactions and metabolites"`

Reply:
Addressed in https://github.com/MetabolicAtlas/MetabolicAtlas/pull/830/commits/26a6f3345c95a9162837579b02a0d04ce81fdfe3

Thanks again!